frontend/src/component/admin/apiToken/ProjectsList/ProjectsList.test.tsx

```diff
@@ -22,16 +22,16 @@ describe('ProjectsList', () => {
         expect(links[0]).toHaveTextContent('project');
     });
 
-    it('should render asterisk if no projects are passed', async () => {
+    it('should render no text if no projects are passed', async () => {
```

Comment:
What's the `container` in this case? The component is called `ProjectsList` ...

I was confused by this at first, but the projects list here is the projects list _cell_, right? How come we're changing the behavior when you don't pass anything?

Reply:
This was the place where I found an issue with keys for removed projects. I'd like this component to show if we have no projects assigned, and not hide that something is wrong.

Reply:
Right, so what you're saying is that the case where we have no projects in the output means that the API token doesn't work on any projects? Are you sure about that? 

Because I think that if you have one of those ancient keys from before we introduced the new [API key format](https://docs.getunleash.io/reference/api-tokens-and-client-keys#format) (in Unleash 4.3), then you won't have any project data associated with it, but it'll still work for all projects.

So I'm not sure this is the right assumption.

Reply:
Ok, I checked that. Btw, that's Unleash 4.1:

![Screenshot from 2024-07-09 11-32-52](https://github.com/Unleash/unleash/assets/2625371/6237c2cb-f294-4927-b35b-ecc18b0effc1)

Reply:
Oh, look, the sidebar is back! Or maybe it's back now, then? 🤔 

Anyway, what did you find when you checked it? That table doesn't show projects, but if you upgrade Unleash to a whenever we introduced the projects column in the table, what happens?